src/document/change/change.ts

```diff
@@ -145,9 +156,12 @@ export class Change<P extends Indexable> {
   } {
     const changeOpInfos: Array<OperationInfo> = [];
     const reverseOps: Array<HistoryOperation<P>> = [];
-    if (process.env.NODE_ENV !== 'production' && this.operations.length) {
+
+    // TODO(chacha): Remove the below logic after implementing undo/redo feature in the devtools.
+    if (process.env.NODE_ENV === 'development' && this.operations.length) {
```

Comment:
Currently, operations are being sent through DevTools and accumulated from the root too. Is there a specific reason for this approach? Could we possibly allow DevTools to accumulate operations instead of needing to handle them from the root?

And I think we need a process guard. Please refer to the PR below and reflect.
https://github.com/yorkie-team/yorkie-js-sdk/pull/778

Reply:
@hackerwins The operation that accumulates in the root is used for `whiteboard.html`. Once this feature is moved to Devtools, it will be removed (to be done via a separate PR).
Is it okay to hold off on the process guard due to the [issues#768 (comment)](https://github.com/yorkie-team/yorkie-js-sdk/issues/768#issuecomment-2062889039)?

Reply:
@chacha912 I think it would be good to remove it from root now and when we add the feature to devtools, then add it to whiteboard example as well.